packages/protocol/migrations/14_epoch_rewards.ts

```diff
@@ -3,10 +3,13 @@ import { deploymentForCoreContract } from '@celo/protocol/lib/web3-utils'
 import { config } from '@celo/protocol/migrationsConfig'
 import { toFixed } from '@celo/utils/lib/fixidity'
 import { EpochRewardsInstance } from 'types'
+const truffle = require('@celo/protocol/truffle-config.js')
 
-const initializeArgs = async (): Promise<any[]> => {
+const initializeArgs = async (networkName: string): Promise<any[]> => {
+  const network: any = truffle.networks[networkName]
   return [
     config.registry.predeployedProxyAddress,
+    network.from,
```

Comment:
`network.from` is validator 0. I think this should be a different key because it's validator key is quite sensitive.

Reply:
Any ideas what else this could be? validator-0 is fairly convenient as that has been the address we use in our tooling to manage test networks (throughout celotool, I believe that's the address that has faucet balances), but agree that we might need to think differently about baklava.

Reply:
It could be any key, including one generated on the fly. You could generate it from the same mnemonic and might create a new `AccountType.ADMIN` https://github.com/celo-org/celo-monorepo/blob/master/packages/celotool/src/lib/generate_utils.ts#L23

Reply:
What do we currently set other things to? I.e. the approver, reserve spender, etc.? I agree validator 0 isn't the most robust choice, but I believe it's currently the standard choice